api/v1alpha1/runner_types.go

```diff
@@ -145,7 +145,7 @@ type RunnerPodSpec struct {
 	HostAliases []corev1.HostAlias `json:"hostAliases,omitempty"`
 
 	// +optional
-	TopologySpreadConstraints []corev1.TopologySpreadConstraint `json:"topologySpreadConstraint,omitempty"`
```

Comment:
Hey! Thanks for the correction.
Could you instead add a new field so that we can safely remove it in a later version (like v1alpha2)?